runtimevar/filevar/filevar.go

```diff
@@ -103,18 +103,11 @@ func (o *URLOpener) OpenVariableURL(ctx context.Context, u *url.URL) (*runtimeva
 	for param := range q {
 		return nil, fmt.Errorf("open bucket %q: invalid query parameter %q", u, param)
 	}
-	return New(mungeURLPath(u.Path, os.PathSeparator), o.Decoder, &o.Options)
-}
-
-func mungeURLPath(path string, pathSeparator byte) string {
-	if pathSeparator != '/' {
+	path := u.Path
+	if os.PathSeparator != '/' {
```

Comment:
I think you can use `filepath.Clean` to simplify this.

Reply:
I don't think that will strip the "/" prefix. It also does a bunch of other things I'm not sure I want.